source/Ship.cpp

```diff
@@ -2498,9 +2502,9 @@ int Ship::Scan()
 					+ Name() + "\" completed its scan of your outfits.", Messages::Importance::High);
 	}
 
-	// Some governments are provoked when a scan is started on one of their ships.
+	// Some governments are provoked when a scan is completed on one of their ships.
 	const Government *gov = target->GetGovernment();
-	if(gov && gov->IsProvokedOnScan() && !gov->IsEnemy(government)
+	if(result && gov && gov->IsProvokedOnScan() && !gov->IsEnemy(government)
```

Comment:
I don't quite understand this. Is `result` only non-zero if a scan has been completed?

Reply:
Yeah exactly, previously this would have made them provoked by starting the scan which is kinda nonsense. Now they first move away.

*Try pressing the scan button from the other side of the system...* remnants: **How dare you? Exterminate!**